nms/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -883,13 +961,20 @@ function validateSubscriberInfo(
   if (info.authOpc && !isValidHex(info.authOpc)) {
     return 'auth opc is not a valid hex';
   }
+  if (info.networkType == undefined || info.networkType.length == 0) {
+    return 'Not a valid Network Type!';
```

Comment:
Please follow the convention on capitalization that's used in the surrounding code

Reply:
Not sure what are you referring to, but I changed it to coreNetworkType.
`if (info.coreNetworkType == undefined || info.coreNetworkType.length == 0)`